bluepyemodel/model/model_configurator.py

```diff
@@ -144,13 +144,21 @@ def get_gene_based_configuration(self):
             version = None
             for k in nexus_keys:
                 if k["name"] == m["name"]:
-                    version = k.get("modelid", None)
+                    version = k.get("modelid") if "modelid" in k else k.get("modelId", None)
+
+            temp_entry = m.get("temperature", None)
```

Comment:
Do you know if icselector might change how they store temperature and jlp correction? I could not find any ic map with temp or ljp

Reply:
I think I got confused with the updated metadata from the actual subcellular resource, the ic mapping file hasn't changed afaik. You're correct about the temperature and ljp, they seem to be absent or removed from the ic map. Given this, should we retain our current approach, where we default to None, or would you prefer to remove them altogether? Otherwise, I think this PR can be closed.

Reply:
I think we can keep this PR. I've seen both modelid and modelId in the ic_map, so this change should definitely be kept. As for the temp and ljp, maybe it is best to also have legacy and up-to-date format, in the case future iterations of ic map, or ic map for other projects end up with temp and ljp